applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.c

```diff
@@ -29,14 +49,9 @@ int ble_hci_vsc_set_radio_high_pwr_mode(bool high_power_mode)
 		LOG_ERR("Unable to allocate command buffer");
 		return -ENOMEM;
 	}
+
 	cp = net_buf_add(buf, sizeof(*cp));
-	if (high_power_mode) {
```

Comment:
I may see this later, but where does one set this if the FEM is not active?

Reply:
Added a note in the h-file: 

> If the nRF21540 is not used, setting max_tx_power to 10 or 20 will result in +3dBm

Reply:
And also added 3dBm as an option in the enum